openbts/tests/sipauthserve_component_tests.py

```diff
@@ -359,6 +359,18 @@ def test_update_nonexistent_subscriber_by_imsi(self):
     self.assertTrue(self.sipauthserve_connection.socket.recv.called)
     self.assertEqual(response.code, SuccessCode.OK)
 
+  def test_delete_subscriber_when_sqlite_unavailable(self):
+    """Testing the subscriber deletion case when OpenBTS reports sqlite is unavailble"""
+    self.sipauthserve_connection.socket.recv.return_value = json.dumps({
+      'code': 503,
+      'data': { 'sip_buddies': 'something bad',
```

Comment:
Not to be too pedantic about this, but could you please make clear what you mean by "something bad" and "this could be ok" here?